src/utils/setDefaultsPTB.m

```diff
@@ -48,6 +48,9 @@
 
     if isfield(cfg, 'audio') && cfg.audio.do
 
+        cfg.audio.devIdx = [];
+        cfg.audio.playbackMode = 1;
+
```

Comment:
here it would not be `fieldToSet.audio.devIdx = [];` instead of `cfg` ?

Reply:
ha yes.

Thanks for spotting that one !! :-)